package.json

```diff
@@ -12,7 +12,8 @@
     "prettier:check": "prettier --check .",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "generate:changelog": "auto-changelog"
```

Comment:
either use $(npm bin) or npx, but i'm not sure that it will work like that as the auto-changelog is not installed globally

Reply:
yes nice catch you're right i'll fix it